omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/scripts/check_token_using_endpoint.lua.erb

```diff
@@ -0,0 +1,24 @@
+ngx.req.read_body()
+local res
+
+if ngx.var.request_method == "POST" then
+  meth = ngx.HTTP_POST
+else
+  meth = ngx.HTTP_GET
```

Comment:
Sorry, I'm not completely understanding the responses to this.  Given this code, what will happen if the method is DELETE and how does that fit with the goal of it being generic.